types/node/v8.d.ts

```diff
@@ -445,7 +445,7 @@ declare module "v8" {
      * @since v8.0.0
      * @param buffer A buffer returned by {@link serialize}.
      */
-    function deserialize(buffer: NodeJS.TypedArray): any;
+    function deserialize(buffer: Buffer | NodeJS.TypedArray | DataView): any;
```

Comment:
```suggestion
    function deserialize(buffer: NodeJS.ArrayBufferView): any;
```
`NodeJS.ArrayBufferView` is the canonical alias.